### PR TITLE
PYTHON-3439 Changelog for 1.1.0

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+Changes in Version 1.1.0
+------------------------
+
+- Use ``boto3`` to fetch on-demand credentials, expanding and standardizing authorization capabilities.
```
Comment (ShaneHarvey) [inline CHANGELOG.rst:7]: Can we mention the newly support EKS feature?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+- Enable opt-in caching of fetched on-demand credentials, to prevent rate
+limiting.
+- Make the ``pymongo.auth.aws_temp_credentials`` function public so it
```
Comment (ShaneHarvey) [inline CHANGELOG.rst:10]: pymongo.auth.aws_temp_credentials -> pymongo_auth_aws.auth.aws_temp_credentials

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+can be used in ``pymongocrypt``.
```
Comment (ShaneHarvey) [inline CHANGELOG.rst:11]: Missing 2 leading spaces.

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (blink1073) [reply CHANGELOG.rst]: Done

File: CHANGELOG.rst (hunk 2 above)
Comment (blink1073) [reply CHANGELOG.rst]: Fixed

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**